chrome_extension/styles.css

```diff
@@ -607,7 +607,7 @@ header {
         font-size: 24px;
         color: #10174C;
         border: none;
-        width: 40px;
+        width: 60px;
         text-align: right;
```

Comment:
All these CSS updates are pushing the colour input to below the visible area.
Adding `white-space: nowrap;` to the .fieldInput will make sure it stays where it should.